data-import/src/main/resources/folijet/data-import/features/instance-identifier-match.feature

```diff
@@ -0,0 +1,384 @@
+                  },
+                  {
+                    "label": "indicator2",
+                    "value": "*"
+                  },
+                  {
+                    "label": "recordSubfield",
+                    "value": "a"
+                  }
+                ],
+              "qualifier": {
+                "qualifierType": "BEGINS_WITH",
+                "qualifierValue": "(AMB"
+              }
+            },
+            "matchCriterion": "EXACTLY_MATCHES",
+            "existingMatchExpression": {
+              "dataValueType": "VALUE_FROM_RECORD",
+              "fields": [
+                {
+                  "label": "field",
+                  "value": "instance.identifiers[].value"
+                },
+                {
+                  "label": "identifierTypeId",
+                  "value": "7e591197-f335-4afb-bc6d-a6d76ca3bace"
+                }
+              ],
+              "qualifier": {}
+            }
+          }
+        ],
+        "deleted": false,
+        "parentProfiles": [],
+        "childProfiles": [],
+        "hidden": false
+      },
+      "addedRelations": [],
+      "deletedRelations": []
+    }
+    """
+    When method POST
+    Then status 201
+    * def matchProfileId = response.id
+
+    # Create a new "Field mapping profile" from the "Settings" page
+    Given path 'data-import-profiles/mappingProfiles'
+    And headers headersUser
+    And request
+    """
+{
+  "profile": {
+    "name": "#(name)",
+    "description": "#(name)",
+    "incomingRecordType": "MARC_BIBLIOGRAPHIC",
+    "existingRecordType": "INSTANCE",
+    "mappingDetails": {
+      "name": "instance",
+      "recordType": "INSTANCE",
+      "mappingFields": [
+        {
+          "name": "staffSuppress",
+          "enabled": "true",
+          "path": "instance.staffSuppress",
+          "value": "",
+          "booleanFieldAction": "ALL_FALSE",
+          "subfields": []
+        },
+        {
+          "name": "catalogedDate",
+          "enabled": "true",
+          "path": "instance.catalogedDate",
+          "value": "\"2021-12-04\"",
+          "subfields": []
+        },
+        {
+          "name": "statusId",
+          "enabled": "true",
+          "path": "instance.statusId",
+          "value": "",
+          "subfields": [],
+          "acceptedValues": {
+            "52a2ff34-2a12-420d-8539-21aa8d3cf5d8": "Batch Loaded",
+            "9634a5ab-9228-4703-baf2-4d12ebc77d56": "Cataloged"
+          }
+        }
+      ]
+    }
+  },
+  "addedRelations": [],
+  "deletedRelations": []
+}
+    """
+    When method POST
+    Then status 201
+    * def mappingProfileId = response.id
+
+    # Create a new "Action profile" from the "Settings" page
+    Given path 'data-import-profiles/actionProfiles'
+    And headers headersUser
+    And request
+    """
+    {
+      "profile": {
+        "name": "#(name)",
+        "description": "#(name)",
+        "action": "CREATE",
```

Comment:
Hello @KyryloShulzhenko, looks like we need the action profile to update the instance, right?